framework/include/wifi_manager/wifi_manager.h

```diff
@@ -386,7 +386,7 @@ wifi_manager_result_e wifi_manager_disconnect_ap(void);
  *     if both SSID and channel are not set in config then it'll return error.
  *     wifi_manager check whether SSID is set by ssid_length in config. In the same way
  *     not checking channel is 0, but checking channel is greater than 0 and less than 15 in 2.5GHz.
```

Comment:
In the code, we checked the channel length with `14`. But in the description it says `15`.
Please check this.

Reply:
the valid range of 2.4G channel is from 1 to 14.